pkg/decision/evaluator/audience.go

```diff
@@ -1,45 +0,0 @@
-/****************************************************************************
```

Comment:
Why deleted this file?

Reply:
this file wasn't being used anywhere throughout the sdk and was throwing syntax errors due to the changes made in this PR.